src/libsystemd/sd-bus/sd-bus.c

```diff
@@ -2933,6 +2933,7 @@ static int process_message(sd_bus *bus, sd_bus_message *m) {
         bus->current_message = m;
         bus->iteration_counter++;
 
+        LOG_CONTEXT_CONSUME_STRV(bus_message_log_context_fields(m));
```

Comment:
This doesn't only apply to the debug log message below. During `process_message()`, all the callbacks registered in the vtable for the service get called as well.

The idea of adding the fields to the context here is to add these fields to any log message that's logged from those callbacks, so that later, we can easily query for all the log messages that happened during processing of a specific D-Bus method by adding a `DBUS_METHOD=Reload` match to journalctl for example.